packages/stack/src/views/Stack/StackView.tsx

```diff
@@ -139,6 +139,11 @@ export default class StackView extends React.Component<Props, State> {
     const isAnimationEnabled = (key: string) => {
       const descriptor = props.descriptors[key] || state.descriptors[key];
 
+      const animation = descriptor?.options?.animation;
+      if (animation && animation === 'none') {
```

Comment:
The `animation` check is unnecessary

```suggestion
      if (animation === 'none') {
```